src/CMakeLists.txt

```diff
@@ -67,6 +67,10 @@ target_include_directories(CabanaMD PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}
 
 target_link_libraries(CabanaMD Cabana::cabanacore Kokkos::kokkos MPI::MPI_CXX dl hwloc)
 
+if(CabanaMD_ENABLE_ArborX)
+  target_link_libraries(CabanaMD ArborX::ArborX)
```

Comment:
I expect you would transitively get it from Cabana.

Reply:
Similarly, I don't think you need to specify the `Kokkos::kokkos` target above

Reply:
Yeah, we should fix that in Cabana.

Reply:
Yeah, we should fix that in Cabana.